src/mcts/params.cc

```diff
@@ -60,6 +60,10 @@ const OptionId SearchParams::kCpuctBaseId{
     "higher growth of Cpuct as number of node visits grows."};
 const OptionId SearchParams::kCpuctFactorId{
     "cpuct-factor", "CPuctFactor", "Multiplier for the cpuct growth formula."};
+const OptionId SearchParams::kNewUEnabledId{
+    "new-u-enabled", "NewUEnabled", 
```

Comment:
Neither flag name, nor description (what corresponds to true and what to false?) are completely clear.

Maybe `--parent-based-u`? `--include-initial-visit-to-u`? `--u-parent-visits`?